.github/workflows/tests.yml

```diff
@@ -0,0 +1,102 @@
+---
+name: Terraform modules tests
```

Comment:
not sure about dynamic population, afraid that it wound overcomplicate things instead of just having 2 separate workflows (but also not sure how much duplication it would be in that case)